### ISSUE TITLE
Detect menu click event 

### ISSUE THREAD

**Issue by Krb686** [joeldbirch/superclick#9]
Probably not necessarily an issue with your code, however I am wondering if a method is exposed or if there is a way to acquire an on click event for a menu.

My problem is that my application actually uses two menus side-by-side. So clicking across the boundary from one menu to the other results in two sub-menus being open and overlapping eachother.  It could probably be solved if I could get an on click event and then call otherMenu.hide() from its handler

**Comment by Krb686** [comment]
Though now that I think about it, I suppose this could be done with basic jQuery events bound to the link element

**Comment by Krb686** [comment]
Alright, so I have event handlers working...but superclick's hide method does not always work.

I have:

```
var generalMenu = $('ul.generalMenus').superclick({...

var otherMenu = $('ul.otherMenu').superclick({...
```

Later on in the event handler for `otherMenu`:

```
generalMenu.children('li:first').superclick('hide');
```

which most of the time, does nothing.

**Comment by Krb686** [comment]
This issue was solved by updating from jquery-1.7.2 to jquery-1.8.3.
